index.html

```diff
@@ -16,16 +17,123 @@
 </head>
 
 <body>
-  <div class="container">
+  <div class="container-fluid d-flex justify-content-center sticky-top nav-bg">
     <div class="row">
-      <div class="col-sm-12">
-        Good Luck!
+      <div class="col-lg-12 text-center nav-bg">
+        <h1>SpaceWalkers</h1>
+        <nav class="d-flex justify-content-between">
+          <a href="">Home</a>
+          <a href="">About Us</a>
+          <a href="">Blog</a>
+          <a href="">Contact</a>
+          <a href="">Sign In</a>
+        </nav>
+      </div>
+    </div>
+  </div>
+
+  <div class="container col-lg-10 mb-5">
+    <div id="carouselExampleControls" class="carousel slide" data-ride="carousel">
```

Comment:
Remember to always use class container, then row, and sizing after. It's a readability thing. However, your carousel is very well executed here.